fakelibusb_devices.go

```diff
@@ -15,175 +15,194 @@
 package gousb
 
 // fake devices connected through the fakeLibusb stack.
-var fakeDevices = []*DeviceDesc{
+type fakeDeviceDef struct {
```

Comment:
you could use the same fakeDevice struct that fakelibusb_test.go declares. It has an extra "alt" field that would go unused, that's fine.

Reply:
done.